<tt>
When&nbsp;debugging&nbsp;I&nbsp;always&nbsp;get&nbsp;an&nbsp;exception&nbsp;when&nbsp;I&nbsp;attempt&nbsp;to&nbsp;step&nbsp;into&nbsp;a&nbsp;method&nbsp;(F11)&nbsp;which&nbsp;itself&nbsp;has&nbsp;a&nbsp;method&nbsp;in&nbsp;the&nbsp;parameters. &nbsp;Not&nbsp;sure&nbsp;if&nbsp;that&amp;#39;s&nbsp;bad&nbsp;practice&nbsp;or&nbsp;not,&nbsp;but&nbsp;I&nbsp;do&nbsp;it...&lt;br&gt;&lt;br&gt;For&nbsp;example,&nbsp;if&nbsp;I&nbsp;try&nbsp;to&nbsp;step&nbsp;into:&lt;br&gt;<br>
&lt;br&gt;MyMethod(GetString(Resource.@string.mystring)&lt;br&gt;&lt;br&gt;It&nbsp;bombs...&lt;br&gt;&lt;br&gt;GetMembers(GetMyMemberID(&amp;quot;Dan&amp;quot;))&nbsp;would&nbsp;also&nbsp;throw&nbsp;an&nbsp;error...&lt;br&gt;&lt;br&gt;It&nbsp;seems&nbsp;like&nbsp;any&nbsp;method&nbsp;or&nbsp;property&nbsp;that&nbsp;is&nbsp;contained&nbsp;within&nbsp;the&nbsp;parameters&nbsp;of&nbsp;another&nbsp;method&nbsp;bombs&nbsp;when&nbsp;you&nbsp;try&nbsp;to&nbsp;step&nbsp;into&nbsp;it,&nbsp;running&nbsp;the&nbsp;app&nbsp;without&nbsp;debugging&nbsp;works&nbsp;fine.&lt;br&gt;<br>
&lt;br&gt;Is&nbsp;that&nbsp;a&nbsp;bug?&lt;br&gt;&lt;br&gt;Dan&lt;br&gt;&lt;br&gt;&lt;br&gt;<br>

</tt>
